packages/@storefront/flux-capacitor/src/core/actions/payloads.ts

```diff
@@ -157,6 +157,12 @@ namespace Payload {
     isFetchingForward?: boolean;
     isFetchingBackward?: boolean;
   }
+
+  export interface Sort {
+    labels?: string[];
+    options: { field: string, descending?: boolean }[];
```

Comment:
The object type should be given a name and made into its own interface.

Reply:
I believe this interface actually exists elsewhere within `FluxCapacitor`. Will find and apply.